README.md

```diff
@@ -1,56 +1,39 @@
 # Open MCT
 
-Open MCT is a next-generation mission control framework being developed at the NASA Ames Research Center in Silicon Valley. Web-based, for desktop and mobile.
+Open MCT is a next-generation mission control framework for visualization of data on desktop and mobile devices. It is developed at NASA's Ames Research Center, and is being used by NASA for data analysis of spacecraft missions, as well as planning and operation of experimental rover systems. As a generalizable and open source framework, Open MCT could be used as the basis for building applications for planning, operation, and analysis of any systems producing telemetry data.
 
-Please visit our [Official Site](https://nasa.github.io/openmct/)
+Please visit our [Official Site](https://nasa.github.io/openmct/) and [Getting Started Guide](https://nasa.github.io/openmct/getting-started/)
 
-##How is NASA Using Open MCT?
+## See Open MCT in Action
 
-Software based on Open MCT is being used for mission planning and operations in the lead up to the [Resource Prospector mission](https://www.nasa.gov/resource-prospector) and at NASA's Jet Propulsion Laboratory to view data from the Curiosity Rover.
-
-##Features
-
- - Support planning and operations of any system that produces telemetry
- -  Support space missions
- - Visualize Data
-   - Streaming Data
-   - Historical Data
-   - Imagery
-   - Timelines
-   - Procedures
-   - etc.
-
-##See Open MCT in Action
-
-Do you want to see how Open MCT work? Try our demo.
-[LIVE DEMO](https://openmct-demo.herokuapp.com/)
+Try Open MCT now with our [live demo](https://openmct-demo.herokuapp.com/).
 ![Demo](https://nasa.github.io/openmct/static/res/images/Open-MCT.Browse.Layout.Mars-Weather-1.jpg)
 
-##Getting Started
+## Documentation
 
-Here is how to run Open MCT in you local machine.
+Check our [Documentation on website](https://nasa.github.io/openmct/documentation/).
```

Comment:
Let's reword slightly. 

    Documentation is available on the [Open MCT website](https://nasa.github.io/openmct/documentation/)'